.gitignore

```diff
@@ -13,4 +13,6 @@
 
 *.obj
 
-*.qm
\ No newline at end of file
+*.qm
+
+src/boardgameprojectfile/flatbuffers/*.*
```

Comment:
This line is not needed - we are not going to include flatbuffers into our source tree.